specs/PermissionManagement.md

```diff
@@ -503,6 +510,12 @@ namespace Microsoft.Web.WebView2.Core
         // developers use the [Local Font Access API](https://wicg.github.io/local-font-access/)
         // to query the system fonts available for styling web content.
         LocalFonts = 10,
+
+        // Indicates permission for a site to use system exlusive messages to access
+        // Musical Instrument Digital Interface (MIDI) devices. Permission is requested
+        // when developers use the [Web MIDI API](https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API)
+        // to request system exclusive MIDI access.
+        MidiSystemExclusiveAccess = 11,
```

Comment:
Maybe we should call it "MidiSystemExclusiveMessageAccess" since it controls access to "MIDI system exclusive messages", not to "exclusive MIDI access".

Reply:
Yes, lets change it to this.